### ISSUE TITLE
[Bug] Fullscreen PWAs are put in PIP mode when pressing home

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#17967]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### Steps to reproduce
- open a pwa which runs in fullscreen mode. (I tested with https://paperplanes.world/)
- press home

### Expected behavior
- the pwa (Fenix) should be put in background

### Actual behavior
- the pwa is put in pip mode - small window floating on the homescreen

https://user-images.githubusercontent.com/11428869/130804428-cd853a8c-3a98-422c-8edd-15245839d422.mp4

### Device information

* Device vendor / model and Android version: Device independent.
* Firefox for Android version: All Fenix versions are affected.

**Comment by Mugurell** [comment]
Currently we're only checking [if the page is in fullscreen](https://github.com/mozilla-mobile/android-components/blob/b266ac3d16e15cabc969cfe33c9cf965f557e257/components/feature/session/src/main/java/mozilla/components/feature/session/PictureInPictureFeature.kt#L46)
What Chrome, Fennec and other browsers do is checking if there is media playing in fullscreen.
We should do the same.

**Comment by CharlesAdam** [comment]
Hello @Mugurell, I would like to take this issue, however, since this issue involves the [ **android-components repo** ](https://github.com/mozilla-mobile/android-components) should and issue be opened on its repo?

**Comment by Mugurell** [comment]
> Hello @Mugurell, I would like to take this issue, however, since this issue involves the [ **android-components repo** ](https://github.com/mozilla-mobile/android-components) should and issue be opened on its repo?

Indeed, a new issue must be opened on AC with the AC PR targetting that ticket.
After merging the patch there this issue can be verified if fixed.

Do you want to open an issue similar to this on AC or I can do it...?

**Comment by CharlesAdam** [comment]
> 
> 
> > Hello @Mugurell, I would like to take this issue, however, since this issue involves the [ **android-components repo** ](https://github.com/mozilla-mobile/android-components) should and issue be opened on its repo?
> 
> Indeed, a new issue must be opened on AC with the AC PR targetting that ticket.
> After merging the patch there this issue can be verified if fixed.
> 
> Do you want to open an issue similar to this on AC or I can do it...?

Please do it, I'm still getting used to Github.

**Comment by Mugurell** [comment]
@CharlesAdam Filed https://github.com/mozilla-mobile/android-components/issues/9698
If you plan to work on that please leave a comment in the issue so I can assign it to you.

**Comment by Mugurell** [comment]
We should rather check if this is still an issue.

**Comment by SoftVision-LorandJanos** [comment]
I managed to reproduce the issue as described, using a Release build from February (85.1.2), with a Xiaomi Mi11 (Android 11) device.

Verified on the following current builds where the issue is no longer reproducible:
- Nightly (93.0a1 - 2021-08-25T05:10:54.365489).
- Beta (92.0.0 beta 6).
- Release (RC 91.3.0).

Device used: Xiaomi Mi11 Lite (Android 11).

As the issue is not reproducible anymore, I will close the ticket as verified fixed, and will remove the QA needed label.
